main/test/flix/Test.Exp.Concurrency.Spawn.flix

```diff
@@ -3,65 +3,65 @@ mod Test.Exp.Concurrency.Spawn {
     import java.lang.NullPointerException
 
     @test
-    def testSpawn01(): Unit \ IO = region rc { spawn () @ rc }
```

Comment:
These tests are a bit weird since `spawn () @ rc    :    Unit \ {}` so they are vulnerable to inlining, making the test a no op